---
date: 2022-06-16T09:50:00
title: Using simulations to assess the performance of genotype-to-phenotype models
  accounting for pleiotropy, variance-controlling loci, and interspecific breeding
  material
authors:
- alipka
authors_display:
- '{{% mention_name "alipka" "Alex Lipka" %}}, Department of Crop Sciences, University
  of Illinois at Urbana-Champaign'
---
Models that reflect the multifaceted contributions of genomic loci have a potential to facilitate unprecedented quantification of the genetic architecture underlying various traits and increase genomic selection (GS) prediction accuracies. To evaluate the performance of such models, simulation studies are essential. Therefore, a R/CRAN package called simplePHENOTYPES developed by the Lipka Lab is first discussed. This package uses real marker data to simulate pleiotropic quantitative trait nucleotides (QTNs) that behave in either an additive, dominance, or epistatic manner. We first demonstrate how simulating traits from simplePHENOTYPES can be used to evaluate the ability of a various multi-trait genome-wide association study (GWAS) models to distinguish between linkage and pleiotropy. We then show how simulations can be used to determine how well statistical approaches used to quantify variance heterogeneity can identify epistasis and genotype-by-environment interactions. We end the presentation by demonstrating a simulation study that evaluates the impact of training set composition from two different species (Miscanthus sinensis and Miscanthus sacchariflorus) on the GS prediction accuracy in an interspecific Miscanthus Ã— giganteus F2 population.
