---
date: 2024-06-13T09:00:00
title: Quantifying evidence for the omnigenic genetic architecture through a simulation
  study.
authors:
- alipka
authors_display:
- '{{% mention_name "alipka" "ALEX LIPKA" %}}, Dept. of Crop Sciences, University
  of Illinois, Urbana-Champaign, IL'
---
Typical GWAS and GS statistical models make assumptions on genetic architecture that were derived from RA Fisherâ€™s milestone 1918 paper. Empirical findings from GWAS and GS support these assumptions, namely that gene effect sizes underlying complex traits become infinitesimally small and that the resulting genetic value converges to a normal distribution. The omnigenic model of Boyle et al. (2017) and Liu et al. (2019) is a prominent example of how findings in molecular and functional biology that have occurred since 1918 can be incorporated into the established framework of complex genetic architecture. This model partitions genes underlying complex traits into a core set that directly controls the trait and a peripheral set that indirectly controls a trait through trans regulation. A refinement of the omnigenic model from Mathieson (2021) postulates that for a given trait, core gene effects are the same across subpopulations, while peripheral gene effect networks change across subpopulations. We hypothesize is that the omnigenic model accurately depicts the genetic architecture across subpopulations. Therefore, the purpose of this ongoing work is to explore how genetic architecture evolves across subpopulations. We are conducting a forward-in-time simulation study to simulate three divergent breeding programs, and then study the evolution of trait genetic architecture. For each generation, traits that are dependent upon additive and epistatic effects of core and peripheral genes are being simulated. The population variances of these effects vary according to a 3x4x4x4 factorial experiment. A combination of GWAS results, variance component estimates, and predictive abilities from multiple genomic prediction models will be used to evaluate the results of this factorial experiment. If this simulation study suggests that the omnigenic model is an accurate depiction of cross-subpopulation genetic architecture, then GWAS and GS models that incorporate features of this model should become more widely used in practice. Such models could result in a refined understanding of genetic architecture and could help optimize breeding values on a species-wide basis. 
